ipatests/test_webui/test_selinuxusermap.py

```diff
@@ -25,22 +25,30 @@
 from ipatests.test_webui.ui_driver import screenshot
 import ipatests.test_webui.data_user as user
 import ipatests.test_webui.data_group as group
+import ipatests.test_webui.data_selinuxusermap as selinux
```

Comment:
`selinux` is bit misleading. `selinuxmap` or `selinuxusermap` seems better

Reply:
Yep, makes sense. Fixed.